packages/auth/express/auth.js

```diff
@@ -1,8 +1,13 @@
 const session = require('express-session')
-const PgSession = require('connect-pg-simple')(session)
 const passport = require('passport')
+const PgSession = require('connect-pg-simple')(session)
+
 const transformUser = require('../lib/transformUser')
 const basicAuthMiddleware = require('./basicAuth')
+const { specialRoles, userIsInRoles } = require('../lib/Roles')
+
+const DEFAULT_MAX_AGE_IN_MS = 60000 * 60 * 24 * 365 // 1 year
+const SHORT_MAX_AGE_IN_MS = 60000 * 60 * 24 * 1 // 1 week
```

Comment:
paranoid me would do the opposite:
`LONG_MAX_AGE_IN_MS= 1 year`
`DEFAULT_MAX_AGE_IN_MS = 1 week`
but your solution is ok aswell.

Reply:
Not giving in to paranoia here.

Reply:
(To be fair: I did have the very same thought. But overcame my bias.)